src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -558,6 +558,11 @@
     <Field type="ulong" contentType="bytes" name="used" label="Used" />
   </Event>
 
+  <Event name="JitRestart" category="Java Virtual Machine, Compiler" label="JIT restart" stackTrace="false" startTime="false" thread="true">
+    <Field type="int" name="freedMemory" label="Freed Memory" />
+    <Field type="ulong" name="codeCacheMaxCapacity" label="CodeCache maximum capacity" />
```

Comment:
Is codeCacheMaxCapacity the current in-use size of the CodeCache, in bytes? It should have the contentType="bytes" in that case. Also "freedMemory" should have the same contentType.

Reply:
Yes, codeCacheMaxCapacity is given in bytes. It specifies the maximum size, not the current in-use size, of the CodeCache. It is the sum over all CodeHeap segments.